locale/ru_RU.json

```diff
@@ -0,0 +1,114 @@
+    "上传": "Загрузить",
+    "双栏pdf": "Двухколоночный PDF",
+    "总结": "Подведение итога",
+    "识别公式": "Распознавание формул",
+    "在这里输入System Prompt...": "Введите здесь системное подсказку...",
+    "加载Prompt模板": "Загрузить шаблон Prompt",
+    "选择Prompt模板集合文件": "Выберите файл с набором шаблонов Prompt",
+    "🔄 刷新": "🔄 Обновить",
+    "从Prompt模板中加载": "Загрузить из шаблона Prompt",
+    "保存/加载": "Сохранить/загрузить",
+    "保存/加载对话历史记录": "Сохранить/загрузить историю диалога",
+    "从列表中加载对话": "Загрузить диалог из списка",
+    "🗑️ 删除": "🗑️ Удалить",
+    "设置文件名: 默认为.json，可选为.md": "Установить имя файла: по умолчанию .json, можно выбрать .md",
+    "设置保存文件名": "Установить имя сохраняемого файла",
+    "对话历史记录": "История диалога",
+    "💾 保存对话": "💾 Сохранить диалог",
+    "📝 导出为Markdown": "📝 Экспортировать в Markdown",
+    "默认保存于history文件夹": "По умолчанию сохраняется в папку history",
+    "微调": "Своя модель",
+    "训练状态": "Статус обучения",
+    "在这里[查看使用介绍](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/%E4%BD%BF%E7%94%A8%E6%95%99%E7%A8%8B#%E5%BE%AE%E8%B0%83-gpt-35)": "[Здесь](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/%E4%BD%BF%E7%94%A8%E6%95%99%E7%A8%8B#%E5%BE%AE%E8%B0%83-gpt-35) можно ознакомиться с инструкцией по использованию",
+    "准备数据集": "Подготовка набора данных",
+    "数据集预览": "Предпросмотр набора данных",
+    "选择数据集": "Выберите набор данных",
+    "上传到OpenAI": "Загрузить в OpenAI",
+    "训练": "Обучение",
+    "文件ID": "Идентификатор файла",
+    "Token 计数: ": "Использованно токенов: ",
+    "，本次对话累计消耗了 ": ", Общая стоимость этого диалога составляет ",
+    "上传到 OpenAI 后自动填充": "Автоматическое заполнение после загрузки в OpenAI",
+    "模型名称后缀": "Суффикс имени модели",
+    "可选，用于区分不同的模型": "Необязательно, используется для различения разных моделей",
+    "训练轮数（Epochs）": "Количество эпох обучения",
+    "开始训练": "Начать обучение",
+    "状态": "Статус",
+    "刷新状态": "Обновить статус",
+    "取消所有任务": "Отменить все задачи",
+    "添加训练好的模型到模型列表": "Добавить обученную модель в список моделей",
+    "高级": "Расширенные настройки",
+    "切换亮暗色主题": "Переключить светлую/темную тему",
+    "实时传输回答": "Передача ответа в реальном времени",
+    "🔄 检查更新...": "🔄 Проверить обновления...",
+    "# ⚠️ 务必谨慎更改 ⚠️": "# ⚠️ ВНИМАНИЕ: ИЗМЕНЯЙТЕ ОСТОРОЖНО ⚠️",
+    "参数": "Параметры",
+    "停止符，用英文逗号隔开...": "Разделительные символы, разделенные запятой...",
+    "用于定位滥用行为": "Используется для выявления злоупотреблений",
+    "用户名": "Имя пользователя",
+    "网络参数": "Параметры сети",
+    "---\n⚠️ 为保证API-Key安全，请在配置文件`config.json`中修改网络设置": "---\n⚠️ Для обеспечения безопасности API-Key, измените настройки сети в файле конфигурации `config.json`",
+    "🔙 恢复默认网络设置": "🔙 Восстановить настройки сети по умолчанию",
+    "未设置代理...": "Прокси не настроен...",
+    "代理地址": "Адрес прокси",
+    "🔄 设置代理地址": "🔄 Установить адрес прокси",
+    "🔄 切换API地址": "🔄 Переключить адрес API",
+    "川虎Chat 🚀": "Чуаньху Чат 🚀",
```

Comment:
I think that "Чуаньху" refers to rivers and lakes in China? It might not accurately convey the original meaning of "川虎"…?

Maybe using "Chuanhu" written in Cyrillic letters would just be fine... 👀@GaiZhenbiao

Reply:
I think using "Chuanhu" is just fine.